ui/src/app/utils/inactivity.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  clearLastActive,
+  getLastActiveEpochMillis,
+  INACTIVITY_CONFIG,
+  setLastActive,
+  setLastActiveNow,
+} from './inactivity';
+
+describe('inactivity last-active accessors', () => {
+  it('should round-trip', () => {
+    setLastActive(123);
+    expect(getLastActiveEpochMillis()).toEqual(123);
+  });
+
+  it('should clear', () => {
+    setLastActive(123);
+    clearLastActive();
+    expect(getLastActiveEpochMillis()).toBeNull();
+  });
+
+  it('should store values as epoch milli strings', () => {
```

Comment:
This test seems like it's validating an implementation detail. Is it needed?

If so, I'd prefer a name like `should store the last active time in the browser's local storage` which describes the side effect on the browser. If in-browser debuggablilty is a test-worthy feature, then the `epoch milli strings` detail may be worth mentioning.

Reply:
yeah that's a good description of what it's doing.

updated and added comments